thunderfish/sortfishtype.py

```diff
@@ -0,0 +1,189 @@
+import numpy as np
+import peakdetection as pkd
+from IPython import embed
+
+
+def type_detector(data, samplerate, thres=0.1, plot_data_func=None):
+
+    """ Detects if fish is pulse or wave by calculating the proportion of the time distance between a peak and its
+     following trough, relative to the time between consecutive peaks.
+
+    :param data: (1-D array) The data to be analyzed (Usually the best window already)
+    :param samplerate: (float). Sampling rate of the data in Hz
+    :param thres: (float or array): a positive number setting the minimum distance between peaks and troughs
```

Comment:
this is only a float, not a an array! Name it pulse_thresh then it is clearer what that threshold is good for.